html/pfappserver/root/src/views/Configuration/firewalls/_components/FormTypeBarracudaNg.vue

```diff
@@ -52,6 +52,12 @@
       :column-label="$i18n.t('Default realm')"
       :text="$i18n.t('The default realm to be used while formatting the username when no realm can be extracted from the username.')"
     />
+
+    <form-group-use-connector namespace="use_connector"
+      :column-label="$i18n.t('Use Connector')"
+      :text="$i18n.t('Use the available PacketFence connectors to connect to this authentication source. By default, a local connector is hosted on this server. Using remote connectors is only supported on a standalone instance at the moment.')"
```

Comment:
```suggestion
      :text="$i18n.t('Use the available PacketFence connectors to connect to this firewall.')"
```

It has to be changed in the other form files too